### PR TITLE
Feature/66 introduce logging config

### LINKED ISSUE #66: Introduce logging config
## Background
There is no configuration for logging in to the current project.

## Feature
Introduce it to provide a more flexible configuration.

### PR DESCRIPTION
**Pull request by miroslavpojer**
- Add new setup method and module for logging.
- Refactored all modules and logging calls.

Closes #66

### CODE REVIEW

**Comment by miroslavpojer** [comment]
Release notes:
- Add a new setup method and module for logging.
- Refactored all modules and logging calls.

File: release_notes_generator/utils/logging_config.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+def setup_logging() -> None:
+    """
+    Set up the logging configuration in the project
+
+    @return: None
+    """
+    # Load logging configuration from the environment variables
+    is_verbose_logging: bool = os.getenv("INPUT_VERBOSE", "false").lower() == "true"
+    is_debug_mode = os.getenv("RUNNER_DEBUG", "0") == "1"
```
Comment (MobiTikula) [inline release_notes_generator/utils/logging_config.py:34]: For me verbose and debug are the same mode for logging. This is little confusing.

Comment (miroslavpojer) [reply release_notes_generator/utils/logging_config.py:34]: - Our action(-s) provides the option to activate verbose logging. The user can decide whether to see the more detailed output in the step output.
- GitHub provides the feature to re-run the job with a debug flag. This is controlled by the RUNNER_DEBUG environment variable. Here, we are checking the value of this variable to automatically provide more detailed output.

So the user of our GH action will not be confused.

**Comment by miroslavpojer** [comment]
> If I understand it correctly the issue was with not having a Handler, which caused not logging into CI.

Yes, from the experiment, this was the source of problems.

**Review by MobiTikula — APPROVED** [review]

**Pull request merged**